src/commands/basemaps-github/create-mapsheet.ts

```diff
@@ -0,0 +1,109 @@
+import { ConfigBundled, ConfigProviderMemory } from '@basemaps/config';
+import { Bounds, EpsgCode } from '@basemaps/geo';
+import { fsa } from '@chunkd/fs';
+import { command, option, optional, string } from 'cmd-ts';
+import * as fgb from 'flatgeobuf/lib/mjs/geojson.js';
+import { FeatureCollection, MultiPolygon } from 'geojson';
+
+import { CliInfo } from '../../cli.info.js';
+import { logger } from '../../log.js';
+import { registerCli, verbose } from '../common.js';
+
+interface Output {
+  sheetCode: string;
+  files: string[];
+}
+
+export const CommandCreateMapSheetArgs = {
+  verbose,
+  path: option({
+    type: string,
+    long: 'path',
+    description: 'Path of flatgeobuf, this can be both a local path or s3 location',
+  }),
+  config: option({
+    type: string,
+    long: 'config',
+    description: 'Path of basemaps config json, this can be both a local path or s3 location',
+  }),
+  output: option({
+    type: string,
+    long: 'output',
+    description: 'Output of the mapsheet file',
+  }),
+  include: option({
+    type: optional(string),
+    long: 'include',
+    description: 'Include the layers with the pattern in the layer name.',
+  }),
+  exclude: option({
+    type: optional(string),
+    long: 'exclude',
+    description: 'Exclude the layers with the pattern in the layer name.',
+  }),
+};
+
+export const basemapsCreateMapSheet = command({
+  name: 'bm-create-mapsheet',
+  version: CliInfo.version,
+  description: 'Create a cog mapsheet from provided flatgeobuf',
+  args: CommandCreateMapSheetArgs,
+  async handler(args) {
+    registerCli(this, args);
+    const path = args.path;
+    const config = args.config;
+    const outputPath = args.output;
+
+    const include = args.include ? new RegExp(args.include.toLowerCase(), 'i') : undefined;
+    const exclude = args.exclude ? new RegExp(args.exclude.toLowerCase(), 'i') : undefined;
```

Comment:
There shouldn't be any need for `toLowerCase` if you're already using case-insensitive search.